tests/main/lxd/task.yaml

```diff
@@ -126,7 +139,16 @@ execute: |
     lxd.lxc exec my-nesting-ubuntu -- snap set lxd waitready.timeout=240
     lxd.lxc exec my-nesting-ubuntu -- lxd waitready
     lxd.lxc exec my-nesting-ubuntu -- lxd init --auto
-    lxd.lxc exec my-nesting-ubuntu -- lxd.lxc launch "ubuntu:${VERSION_ID}" my-inner-ubuntu
+
+    (
+        . /etc/os-release
+        # TODO:2004: remove version check once the image for ubuntu 20.04 is published
+        if [ "$VERSION_ID" == 20.04 ]; then
+            # shellcheck disable=SC2031
+            VERSION_ID=19.10
+        fi
```

Comment:
Should we add a check that fails when 20.04 image becomes available?
```
# this will fail once ubuntu:20.04 images are available
not lxd.lxc image info ubuntu:20.04
```